src/components/layout/navigation/navigation.tsx

```diff
@@ -230,7 +230,7 @@ const Navigation: React.FC<NavigationProps> = ({
     <NavigationBackground>
       <Grid center>
         <GridItem xs={0} md={1} />
-        <GridItem xs={12} md={10} noGap>
+        <GridItem xs={12} md={11} noGap>
```

Comment:
<img width="1097" alt="Bildschirm­foto 2023-03-01 um 17 19 09" src="https://user-images.githubusercontent.com/57712895/222199355-232c6a91-127b-4c0d-85ae-ee6dbaa0bb68.png">

There is something wrong with the alignment. Maybe it makes sense to move the navigation changes to an extra PR? I can not estimate how complex they are

Reply:
Agreed, it is better to do an extra PR, otherwise it is a partial solution.